tests/basic_sparse_grpc_test.sh

```diff
@@ -0,0 +1,159 @@
+#!/bin/bash
+# This test checks that Qdrant answers to all API mentioned in README.md as expected
+
+set -ex
+
+# Ensure current path is project root
+cd "$(dirname "$0")/../"
+
+QDRANT_HOST='localhost:6334'
+
+docker_grpcurl="docker run --rm --network=host -v ${PWD}/lib/api/src/grpc/proto:/proto fullstorydev/grpcurl -plaintext -import-path /proto -proto qdrant.proto"
+
+$docker_grpcurl -d '{
+   "collection_name": "test_sparse_collection"
+}' $QDRANT_HOST qdrant.Collections/Delete
+
+$docker_grpcurl -d '{
+   "collection_name": "test_sparse_collection",
+   "sparse_vectors_config": {
+      "map": {
+        "test": { }
+      }
+   }
+}' $QDRANT_HOST qdrant.Collections/Create
+
+$docker_grpcurl -d '{}' $QDRANT_HOST qdrant.Collections/List
+
+$docker_grpcurl -d '{
+  "collection_name": "test_sparse_collection",
+  "wait": true,
+  "ordering": null,
+  "points": [
+    {
+      "id": { "num": 1 },
+      "vectors": {
```

Comment:
can you please explain me what are those 3 layers on `vectors` respectively?

Reply:
surprisingly enough, that's not new. In old version for named vector you would get same. Price we should pay for backward compatibility